osidb/tests/test_endpoints.py

```diff
@@ -1095,6 +1097,7 @@ def test_get_flaw_with_token(
         flaw1 = FlawFactory.build(
             major_incident_state=Flaw.FlawMajorIncident.APPROVED,
             requires_summary=Flaw.FlawRequiresSummary.APPROVED,
+            nist_cvss_validation=Flaw.FlawNistCvssValidation.REQUESTED,
```

Comment:
Nitpick: No need to add `nist_cvss_validation` into this test, as the original test did not mention it.

You test `nist_cvss_validation` in `test_get_flaw()` and that shoud be sufficient.

Reply:
> Nitpick: No need to add `nist_cvss_validation` into this test, as the original test did not mention it.

- This sentence is under 1100 line and it belongs to the `test_get_flaw_with_token()` test.
- In the past, I added `major_incident_state` and `requires_summary` into `test_get_flaw()` and `test_get_flaw_with_token()` because both these tests had contained `is_major_incident` (this was replaced by `major_incident_state)` and `FlawMetaFactory` of the `REQUIRES_SUMMARY` type (this was replaced by  `requires_summary`).

> You test `nist_cvss_validation` in `test_get_flaw()` and that shoud be sufficient.

- None of the tests in `test_endpoints` contained `FlawMeta`'s `NIST_CVSS_VALIDATION,` but I think putting it into one test (e.g. into `test_get_flaw()`) is ok.

Does it make sense to you now? :)

Reply:
Fixed & committed.